SOP/SOP-push.adoc

```diff
@@ -59,61 +464,318 @@ NOTE: It will use the values mapped in the Secret created by the operator with t
+deploymentconfig.apps.openshift.io/example-unifiedpushserver              1          1         1         config,image(postgresql:postgresql:10),image(ups-imagestream:latest),image(ups-oauth-proxy-imagestream:latest)
+deploymentconfig.apps.openshift.io/example-unifiedpushserver-postgresql   1          1         1         image(postgresql:10)
 
-== General procedure
+NAME                                                         DOCKER REPO                                               TAGS      UPDATED
+imagestream.image.openshift.io/ups-imagestream               172.30.1.1:5000/unifiedpush/ups-imagestream               latest    12 days ago
+imagestream.image.openshift.io/ups-oauth-proxy-imagestream   172.30.1.1:5000/unifiedpush/ups-oauth-proxy-imagestream   latest    12 days ago
 
-. Capture a snapshot of the 'UnifiedPush Server' Grafana dashboard and track it over time. The metrics can be useful for identifying performance issues over time.
+NAME                                                                   HOST/PORT                                                                      PATH      SERVICES                                      PORT      TERMINATION   WILDCARD
+route.route.openshift.io/example-unifiedpushserver-unifiedpush-proxy   example-unifiedpushserver-unifiedpush-proxy-unifiedpush.192.168.64.27.nip.io             example-unifiedpushserver-unifiedpush-proxy   <all>     edge/None     None
+----
 
-. Capture application logs for analysis.
-.. Get the pod names by running `oc get pods`
-.. Save the logs by running `oc logs <database-podname> > <filename>.log` for each pod
+=== Optional configurations
 
-NOTE: You are able to the the logs by the Console (OCP UI) as well.
+==== Monitor
 
-. If necessary, recreate the service pod to restore service.
-. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+If the https://github.com/aerogear/unifiedpush-operator#monitoring-service-metrics[Monitoring Service (Metrics)] is enabled for your installation you are able to check its Grafana Dashboard, `UnifiedPush Server`, and the Prometheus Monitoring instance used for it.
+
+==== Backup
 
+. Switch to the UPS namespace by running `oc project <namespace>`. E.g `oc project unifiedpush`
+. Check that link:./deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup[UnifiedPushServerWithBackup CR] is deployed in the same namespace as the operator by running `oc get UnifiedPushServer`. Following the expected result.
++
+NOTE: Just one kind of UnifiedPushServer CR can be applied, however, if the backup service is enable for your installation then it means that it is using the ink:./deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup[UnifiedPushServerWithBackup CR].
++
+[source,shell]
+----
+$ oc get UnifiedPushServer
+NAME                        AGE
```

Comment:
If we address this suggestion we are not showing an example of the expected result at all.

Reply:
I should have explained it better: the suggestion to remove this line (and the line below it) is coupled with the suggestion on the line above, where I'm suggesting to run `oc get ups -o yaml` rather than `oc get ups`. I think that's a better command to run in this case, since `oc get ups` doesn't give any information about backups, and that appears to be what this section of the SOP is about. Does that make sense?

I think the confusion here is because I can't do "suggestions" that operate on multiple existing lines (I can insert multiple new lines in a suggestion on one line though).

Reply:
The point is that the idea here is:
1. Check if the object is applied 
2. If the object is applied then check its values 

It is easier for who never ever saw the project than check the value and it does not return anything.